### ISSUE TITLE
Accessibility reading list

### ISSUE THREAD

**Issue by TheAdmiral** [Heydon/AMA#1]
Hello!

Apart from your own books, who is writing well on web accessibility? Is there a good reading list out there?

**Comment by Heydon** [comment]
Hi @TheAdmiral. 

I recommend you read Adrian Roselli's blog (https://adrianroselli.com/), and Scott O'Hara goes into a lot of detail too (https://www.scottohara.me/). The book _Accessibility For Everyone_ by Laura Kalbag is very comprehensive, and I edited _Form Design Patterns_ by Adam Silver, which looks at forms specifically with a focus on accessibility and inclusion.

If you would like to read more of my technical solutions to design patterns (like those I covered in _Inclusive Components_), I wrote up a whole load for the BBC's GEL (https://bbc.github.io/gel/).

_A Web For Everyone_ is co-authored by my former colleague, Sarah Horton and is excellent for an introduction to a UX perspective on accessibility.
